resources/generate_book.py

```diff
@@ -462,6 +490,48 @@ def make_book(rm: ResourceManager, i18n: I18n, local_instance: bool = False):
+            image('tfc:textures/gui/book/tutorial/old_cow.png', text_contents='This bull is old and cannot breed, because of its faded coat and grey, unseeing eyes.'),
+            text('Livestock can be fed to raise $(thing)familiarity$(). Each animal has a set of food that it prefers to eat. To feed an animal, $(item)$(k:key.sneak)$() and $(item)$(k:key.use)$() with food in your hand.'),
+            image('tfc:textures/gui/book/tutorial/unfamiliarized_pig.png', text_contents='Livestock have a $(thing)familiarity indicator$() that shows how familiar they are with you. It is visible when holding $(item)$(k:key.sneak)$() and looking at the animal.'),
+            image('tfc:textures/gui/book/tutorial/no_familiarity_decay_pig.png', text_contents='Familiarity decays each day by a little, unless it reaches a certain amount, indicated by the heart being outlined in white.'),
+            image('tfc:textures/gui/book/tutorial/familiarity_limit_pig.png', text_contents='Adult livestock cannot be familiarized above a certain amount, indicated by the red outlined heart. Babies can reach up to 100% familiarity.'),
+            text('$(thing)Mammals$() are livestock that experience $(thing)pregnancy$(). Adult mammals that are above 30% familiarity and have been fed that day will mate, given that they are of opposite genders and near each other. The female animal will become $(thing)pregnant$(), which causes it to spawn children a set number of days after fertilization.'),
+            text('$(thing)Wooly Animals$() are mammals that can be $(thing)sheared$() if they are adults and familiar enough to you.'),
+            text('$(thing)Dairy Animals$() are mammals that make $(thing)milk$(). Female dairy animals can be clicked with a bucket to obtain milk.'),
+            crafting('tfc:crafting/nest_box', title='Nest Box', text_contents='$(thing)Oviparous Animals$() are not mammals, and instead produce children by laying $(item)eggs$(). They need a $(thing)nest box$() to lay eggs, which they are capable of locating on their own.'),
+            heat_recipe('tfc:heating/cooked_egg', '$(item)Eggs$() can be cooked or boiled for food. Male oviparous animals can fertilize females, which causes the eggs in the nest box to be fertilized. The eggs will say how long until they are ready to hatch.'),
+            leather_knapping('tfc:leather_knapping/saddle', '$(thing)Horses$() are mammals that can be ridden when tamed. Horses become rideable after reaching 15% familiarity. They need a $(thing)saddle$() to ride, which can be $(thing)knapped$()'),
+            text('The next few pages will cover different types of animals.$(br)For information on how rainfall and temperature work, see the $(l:the_world/climate#rainfall)rainfall$() and $(l:the_world/climate#temperature)temperature$() pages.'),
```

Comment:
This isn't what I meant. The point is, if you see the word "rainfall", you can click on it and it will link to the rainfall. Having a "rainfall go here" link seventeen pages back doesn't mean much when you look up "Pig" and have to click around from there.

In all the following animal pages, replace *each instance* of "temperature" and "rainfall" with links to the respective anchors.